### PR TITLE
Submitting Party Email Optional

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* /bcgov/entity#16230

*Description of changes:*

- Changes email validation rules to make email optional
- Adds additional tests for validating email of submitting party

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

- Before:

image

- After:

image

image

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1338-mowohsfd.web.app

File: ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue [hunk 1]
```diff
@@ -219,7 +219,7 @@ export default defineComponent({
       isStringOrNumber,
       required,
       isNumber,
-      isEmail,
+      isEmailOrEmpty,
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: Maybe `isEmailOptional` would be a better name.

**Review by dimak1 — APPROVED** [review]
Looks good, thanks for the tests and screenshots!

**Review by cameron-eyds — APPROVED** [review]
LG 👍 
Good to go after addressing Dimas comment

File: ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue (hunk 1 above)
Comment (orelbn) [reply ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue]: I am good with that! ill make the change

**Review by doug-lovett — APPROVED** [review]
Huge! No additional comments.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1338-mowohsfd.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1338-mowohsfd.web.app

**Pull request merged**